pkg/omf/functions/util/omf.check.version.fish

```diff
@@ -0,0 +1,3 @@
+function omf.check.version -a min_version -a current_version
+  test (echo "$min_version"\n"$current_version" | sort -V | head -n1) = "$min_version"
```

Comment:
Actually I was just checking my changes when you posted your version, so I only saw it when the PR was already updated. Otherwise I might have used your code.
On the upside, after (/ if) my edge-case-failing code is merged, you can override it with yours and get proper credits.